userapp_analog_clock/build_user_app.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
```

Comment:
Why do you have shell script and makefile same time.
I believe something one is enough (preferably Makefile if you don't have to use some complex shell scripting)

Reply:
It's just to have an unified way to build and deploy this application